wiki/Help_Centre/en.md

```diff
@@ -521,17 +536,20 @@ We will need the results from a trace route for command for the following server
 
 To copy the results of the command, simply click and drag from the bottom of the output all the way to the top so that the information turns white, then right click to instantly copy it to your clipboard.
 
-#### I have changed my settings, and now I can't start osu! or it freezes
+#### I received a server-side error when trying to play online!
 
-**Resetting osu! back to its default settings will fix this in most cases.**
+**The servers could be having issues.**
 
-Follow these steps to restore osu! to its default settings:
+Check the [osu! server status page](https://status.ppy.sh/) to see if there are any ongoing issues with the servers.
+
+**This can also happen if you are using a shared internet connection, a proxy, or VPN.**
+
+Try using another proxy or VPN provider - if all else fails, send in a support ticket via mail to [accounts@ppy.sh](mailto:accounts@ppy.sh).
+
+If you are not using any of the above, then you will need to send in a support ticket, as your connection has likely been blacklisted in error.
+
+Tor or networks associated with the Tor relay will not connect to the game servers for the most part. This is to avoid malicious use.
```

Comment:
probably shouldn't make it seem like the entire osu! team sees Tor as a malicious service
```suggestion
Tor or networks associated with the Tor relay will not connect to the game servers for the most part. Although we neither condemn nor endorse the use of Tor, we enforce this policy to avoid any potential bad actors using Tor.
```

Reply:
Very true lol, will change that.